gradle/libs.versions.toml

```diff
@@ -0,0 +1,231 @@
+androidx-activity-ktx = { group = "androidx.activity", name = "activity-ktx", version.ref = "androidxActivity" }
+androidx-appcompat = { group = "androidx.appcompat", name = "appcompat", version.ref = "androidxAppcompat" }
+androidx-appcompat-resources = { group = "androidx.appcompat", name = "appcompat-resources", version.ref = "androidxAppcompat" }
+androidx-camera = { group = "androidx.camera", name = "camera-camera2", version.ref ="androidxCamera" }
+androidx-camera-lifecycle = { group = "androidx.camera", name = "camera-lifecycle", version.ref ="androidxCamera" }
+androidx-camera-view = { group = "androidx.camera", name = "camera-view", version.ref ="androidxCamera" }
+androidx-cardView = { group = "androidx.cardview", name = "cardview", version.ref = "androidxCardview" }
+androidx-compose-foundation = { group = "androidx.compose.foundation", name = "foundation" }
+androidx-compose-foundation-layout = { group = "androidx.compose.foundation", name = "foundation-layout" }
+androidx-compose-material = { group = "androidx.compose.material", name = "material" }
+androidx-compose-material3 = { group = "androidx.compose.material3", name = "material3", version.ref ="androidxComposeMaterial3" }
+androidx-compose-runtime = { group = "androidx.compose.runtime", name = "runtime" }
+androidx-compose-runtime-livedata = { group = "androidx.compose.runtime", name = "runtime-livedata" }
+androidx-compose-testManifest = { group = "androidx.compose.ui", name = "ui-test-manifest" }
+androidx-compose-tooling = { group = "androidx.compose.ui", name = "ui-tooling" }
+androidx-compose-ui = { group = "androidx.compose.ui", name = "ui" }
+androidx-compose-ui-graphics = { group = "androidx.compose.ui", name = "ui-graphics" }
+androidx-compose-ui-test = { group = "androidx.compose.ui", name = "ui-test-junit4" }
+androidx-compose-ui-text = { group = "androidx.compose.ui", name = "ui-text" }
+androidx-compose-ui-tooling-preview = { group = "androidx.compose.ui", name = "ui-tooling-preview" }
+androidx-compose-ui-unit = { group = "androidx.compose.ui", name = "ui-unit" }
+androidx-constraintLayout = { group = "androidx.constraintlayout", name = "constraintlayout", version.ref = "androidxConstraintlayout" }
+androidx-constraintLayout-compose = { group = "androidx.constraintlayout", name = "constraintlayout-compose", version.ref ="androidxConstraintlayoutCompose" }
+androidx-core = { group = "androidx.core", name = "core", version.ref = "androidxCore" }
+androidx-core-ktx = { group = "androidx.core", name = "core-ktx", version.ref = "androidxCore" }
+androidx-coreCommon = { group = "androidx.arch.core", name = "core-common", version.ref = "androidxArchCore" }
+androidx-coreRuntime = { group = "androidx.arch.core", name = "core-runtime", version.ref = "androidxArchCore" }
+androidx-coreTesting = { group = "androidx.arch.core", name = "core-testing", version.ref = "androidxArchCore" }
```

Comment:
Minor (🔍): Consider adding the missing `arch` keyword to all, the `coreCommon`, `coreRuntime` and `coreTesting` dependencies, in the end it should like the below:

```toml
androidx-arch-core-common
androidx-arch-core-runtime
androidx-arch-core-testing 
```